### DISCUSSION TITLE
[browser] Alternate (more noticeable) UI for add/update password banner

### DISCUSSION THREAD

**Discussion by AskAlice** [bitwarden/clients#6201] [Password Manager]
### Code Contribution Proposal

Coming from LastPass, I found bitwarden's notifier a bit easy to miss. 

Is there any interest in creating an option to have a floating notifier banner (ie, configurably in the top/bottom left/right corner)?

I also found the color scheme to be a bit tame given the notion that action is required before the next page navigation, which is why at least a box-shadow would help in that regard.

I understand it's important to not inhibit the functionality of.. pretty much any website under the sun that might require interaction in (insert page corner here), however this is why this could be an optional, non-default setting

**Comment by dwbit** [comment]
Hey @AskAlice can you confirm if this is a feature request, or something that you would like to contribute code for?

**Reply by AskAlice** [reply]
Honestly this really depends on the complexity. I am fairly sure that modifying the template's [html](https://github.com/bitwarden/clients/blob/master/apps/browser/src/autofill/notification/bar.html)/[scss](https://github.com/bitwarden/clients/blob/master/apps/browser/src/autofill/notification/bar.scss) would be low effort, however, in terms of settings UI for this, I haven't dug too deeply into what I'd be signing up for to make that happen.

I haven't tried booting up your storybook instance but if stories are necessary I can also provide that.

At the very least I could offer collaboration.

**Reply by dwbit** [reply]
Thanks for confirmation @AskAlice, the team will review and get back to you shortly!

**Comment by dwbit** [comment]
Closing out this older request, feel free to ping if you need anything.
